runtime/libc/wasi_sdk_backing.c

```diff
@@ -1104,7 +1104,16 @@ wasi_errno_t wasi_snapshot_preview1_random_get(
     wasi_size_t buf_baseretptr,
     wasi_size_t buf_len
 ) {
-    wasi_unsupported_syscall(__func__);
+    srandom(time(NULL));
```

Comment:
Bad practice--we should do an if and initialize using `/dev/urandom`

Reply:
I added support for `/dev/urandom` and kept the existing implementation in place as a fallback for more minimal POSIX environments. I modified the behavior to only seed on the first invocation.